lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -586,6 +586,15 @@ module.exports = class MethodMappings {
         async clearElementValue(webElementOrId) {
           const element = this.getWebElement(webElementOrId);
           await element.clear();
+          try {
+            const value = await element.getProperty('value');
+            if (isString(value) && value.length !== 0) {
+              const backArr = Array(value.length).fill(Key.BACK_SPACE);
```

Comment:
A better way to do this would be:
```suggestion
              const backArr = value.split('').map(() => Key.BACK_SPACE)
```

Reply:
sure doing the changes.. apologies

Reply:
@garg3133  i have one query.. why we aren't using Array.fill for this?

Reply:
I was actually not sure about the `Array.fill` method regarding its availability and performance. But if it is available in Node 16 as well, then `Array.fill` can also be used here.